src/views/SpaceCreate.vue

```diff
@@ -263,6 +263,11 @@ async function validateAuthor() {
       return;
     }
 
+    if (props.space.validation.name === 'any') {
+      isValidAuthor.value = true;
+      return;
+    }
```

Comment:
It still goes through if a space has `basic` and `minScore` considering most spaces have this right now. 
```json
"validation": {
"name": "basic",
"params": {}
}
```

Should be fine too, will just reduce few calls to score API ;)

Reply:
Yep good point will change that